jvm/src/vyxal/gen/GenerateDocs.scala

```diff
@@ -57,4 +59,46 @@ private object GenerateDocs:
     SugarMap.trigraphs
       .map { case (key, value) => s"$key -> $value" }
       .mkString("", "\n", "\n")
+
+  def elementTable(): String =
+    val header = "| Symbol | Name | Keywords | Arity | Vectorises | Overloads |"
+    val divider = "| --- | --- | --- | --- | --- | --- |"
+    val contents = StringBuilder()
+    val formatOverload = (overload: String) =>
+      val (args, description) = overload.splitAt(overload.indexOf("->"))
+      val newDesc = description
+        .replace("|", "\\|")
+        .replace("->", "")
+        .stripLeading()
+        .stripTrailing()
+      if args.stripTrailing() == "" then s"`$newDesc`"
+      else
+        s"`${args.stripTrailing().replace("|", "\\|").replace("->", "")}` => `$newDesc`"
+    val addRow = (elem: Element) =>
```

Comment:
Why not put this directly in the foreach?
```suggestion
    def addRow(elem: Element) =
```

Reply:
Because I wanted to write it neatly and separated from the for each. I wanted to give cleaner looking code a try.

Reply:
Hence the lambda. I didn't want to use a def, because it's a temp function

Reply:
not  a problrm anymore because it's in the foreach now